{{ cookiecutter.__project_slug }}/README.md

```diff
@@ -20,6 +28,8 @@ Activate it:
 source venv/bin/activate
 ```
 
+### Install dev Dependencies
```

Comment:
is “dev” case sensitive? if not should probably follow title case and make it “Dev” or “Development”

Reply:
You are right. I changed it. Thanks!